src/mobility/components/BikeStationBottomSheet.tsx

```diff
@@ -29,7 +34,6 @@ type Props = {
 
 export const BikeStationSheet = ({stationId, distance, close}: Props) => {
```

Comment:
Same as the other one - not a part of your changes, but would be nice to update the component name to `BikeStationBottomSheet` to match the file name

Reply:
updated